.ansible/playbook.yml

```diff
@@ -0,0 +1,84 @@
+---
+- hosts: all
+  sudo: true
```

Comment:
http://docs.ansible.com/ansible/become.html
"even though they are deprecated, sudo and su directives, variables and options will continue to work. It is recommended to move to become as they may be retired at one point"

men det behövs för att köra på ansible före 1.9. (debian stable har 1.7 om man inte kör med backports)